docker-assets/run_rails_server

```diff
@@ -24,5 +24,4 @@ function check_svc_status() {
 # Wait for postgres to be ready
 check_svc_status $DATABASE_HOST $DATABASE_PORT
 
-bundle exec rake db:migrate
-bundle exec rails server
+bundle exec rake db:migrate && bundle exec rails server
```

Comment:
When this gets merged we'll want the same change on the approval service as well: https://github.com/RedHatInsights/approval-api

Reply:
@gmcculloug FYI: https://projects.engineering.redhat.com/browse/SSP-1150